backend/dal/Helpers/Migrations/SeedMigration.cs

```diff
@@ -54,39 +54,39 @@ public SeedMigration()
 
         #region Methods
```

Comment:
By default it will look for scripts in `/Migrations/Up` and `/Migrations/Down` and run them during up or down migrations.  Extended this by providing `PreUp`, `PostUp`, `PreDown`, `PostDown`.  Each of these deal with common use-case for running scripts before and after the up and down migrations.